usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -139,6 +139,63 @@ def _outlay_amount_agg_subquery(
     )
 
 
+def _outlay_amount_agg_subquery_no_coalesce(
+    emergency_fund_group_name: str = None,
+    def_codes: Optional[List[str]] = None,
+    award_id_col: Optional[str] = "award_id",
+) -> Subquery:
+    """This function returns a total outlay aggregation
+    where NULLs are not represented as 0s. This function
+    is useful for when you want to show data that doesn't have outlay values
+    as a blank/NULL (i.e. do NOT show “0”).
+    """
+    filters = [filter_by_latest_closed_periods(), Q(award_id=OuterRef(award_id_col))]
+    if emergency_fund_group_name is not None:
+        filters.append(Q(disaster_emergency_fund__group_name=emergency_fund_group_name))
+    if def_codes:
+        filters.append(Q(disaster_emergency_fund__code__in=def_codes))
+
+    return Subquery(
+        FinancialAccountsByAwards.objects.filter(*filters)
+        .values("award_id")
+        .annotate(
+            total_gross_outlay_amount_by_award_cpe=Sum("gross_outlay_amount_by_award_cpe"),
+            total_ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe=Sum(
+                "ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe"
+            ),
+            total_ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe=Sum(
+                "ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe"
+            ),
+        )
+        .filter(
+            *[
+                Q(
+                    Q(total_gross_outlay_amount_by_award_cpe__isnull=False)
+                    | Q(total_ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe__isnull=False)
+                    | Q(total_ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe__isnull=False)
+                )
+            ]
+        )
+        .annotate(
+            sum=Coalesce(
+                "total_gross_outlay_amount_by_award_cpe", 0, output_field=DecimalField(max_digits=23, decimal_places=2)
+            )
+            + Coalesce(
+                "total_ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe",
+                0,
+                output_field=DecimalField(max_digits=23, decimal_places=2),
+            )
+            + Coalesce(
```

Comment:
Like my previous comment about the `Coalesce()` statements? Will these violate AC#5, or does the above filtering prevent that?

Reply:
I see your comment explains that this should handle NULL values. Can you explain how it works?

Reply:
> does the above filtering prevent

The intent is that the filtering prevents it. Let's have a call tomorrow and I can explain.